satzung.md

```diff
@@ -70,7 +70,21 @@ Zu den Aufgaben des Fachschaftsrates gehören insbesondere:
 (5) Näheres regelt die Geschäftsordnung des Fachschaftsrates.
 
 
+## § 4c Fachschaftsrats-Assoziierte
+
+(1) Fachschaftsrats-Assoziierte sind Personen, die den Fachschaftsrat in bestimmten Aufgaben unterstützen. Dies ist nicht möglich bei Ämtern und Aufgaben, die nach der Geschäftsordnung des Fachschaftsrates von dessen Mitgliedern übernommen werden müssen.
```

Comment:
Damit wäre es aber z.B. ok EvaP-Verantwortlichkeiten zu übertragen, was im Sinne des Datenschutzes nicht sinnvoll ist. Mir fällt keine schöne Klausel ein, um das zu Umgehen, außer wenn man das direkt in die Verantwortlichkeit des FSR legt.Vermutlich Ergänzung bei (2)

Reply:
Wenn dir noch eine schöne Formulierung dafür einfällt, schlag sie gerne vor.
Uns ist keine eingefallen. Ich bezweifle, dass wir datenschutzrechtlich einen anderen Status haben als andere Studierende. Gleichzeitig entscheidet ja der FSR, welche Aufgaben er abgibt, da sollte der FSR so verantwortungsbewusst sein, diese Aufgabe nicht abzugeben.
Die bisherige Formulierung unterstreicht nur noch einmal, dass es Ämter gibt, die per Definition nicht von einem Nicht-FSR-Mitglied übernommen werden können

Reply:
Was bzw warum genau wollt ihr eine neue Funktion in die Satzung schreiben? Was ist der Regelungsgehalt?
Das erschließt sich mir noch nicht. Sofern man andere Leute hinzuziehen will, kann man das jederzeit per Beschluss tun.
Etwaiges Fehlverhalten der betreffenden Personen müsste sich der FSR im Zweifel vermutlich selbst zurechnen lassen (mindestens bei natürlichen Personen). Das, was ihr hier versucht zu regeln, sollte bereits über den Gehilfen abgedeckt sein: https://de.wikipedia.org/wiki/Gehilfe (IANAL)

Reply:
Ich finde den Punkt von @cyroxx fair. 
Es geht ja nach wie vor um Dinge die _eigentlich_ der FSR tut, also könnte man das auch einfach in der GO erwähnen, da es ja quasi nur eine Änderung der Arbeitsweise ist